addons/medical_vitals/functions/fnc_updateHeartRate.sqf

```diff
@@ -51,8 +52,11 @@ if IN_CRDC_ARRST(_unit) then {
         if (_painLevel > 0.2) then {
             _targetHR = _targetHR max (80 + 50 * _painLevel);
         };
+        // Increase HR to compensate for low blood oxygen
+        // Increase HR to compensate for higher oxygen demand (e.g. running, recovering from sprint)
+        private _oxygenDemand = _unit getVariable [QGVAR(oxygenDemand), 0];
+        _targetHR = _targetHR + ((97 - _spo2) * 2) + (_oxygenDemand * -1000);
```

Comment:
@BrettMayson I think this is a bit low: with advanced fatigue enabled running on flat ground in VR map, target HR was 76. I understand this is the absolute best case scenario (spo2 variable was at 100%) but I think oxygen demand should have a higher effect on HR. Upping multiplier to -10k resulted in target HR of 96, which still seems low to me, but more fitting.

Reply:
The thing is that this is only simulating one kind of oxygen demand while running, I think if we want the HR to be more realistic, we should have a separate demand as part of spo2 simulation, not tied to this part of fatigue